src/components/CartPage/CartPageContainer.js

```diff
@@ -0,0 +1,17 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { incrementQuantity, decrementQuantity, removeItemFromCart } from '../../actions/actionsCart';
+import CartPage from './CartPage';
+
+const mapStateToProps = ({ reducerCart }) => ({
+  userCart: reducerCart.userCart
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  incrementQuantity: bindActionCreators(incrementQuantity, dispatch),
+  decrementQuantity: bindActionCreators(decrementQuantity, dispatch),
```

Comment:
Can you create container component for CartItem.js to not pass these as props only?

Reply:
ok

Reply:
But I need it in both components

Reply:
i think there is no need to create two containers